### ISSUE TITLE
why is the NT F1 exactly the same as F1 score running this code?

### ISSUE THREAD

**Issue by ZhichaoDuan** [ljynlp/MRN#2]
I cloned this repo and try to replicate the results and I found that each time I evaluate the model performance after training, the NT f1 score and the f1 score are the exact same which is odd..

**Comment by ljynlp** [comment]
There existed such a bug, but I thought it was fixed. I haven't encountered it since I updated the code.
Did you try the bert version?

**Comment by ZhichaoDuan** [comment]
> 

Yeah, I used the bert version but encountered this problem. As for the glove version, I haven't tested yet.
